yellowbrick/cluster/silhouette.py

```diff
@@ -66,6 +66,15 @@ class SilhouetteVisualizer(ClusteringScoreVisualizer):
         The axes to plot the figure on. If None is passed in the current axes
         will be used (or generated if required).
 
+    colormap : str, default: None
+        The name of the matplotlib color map to use for each cluster group. Overrides colors if both are specified.
+
```

Comment:
I think we can safely remove `colormap` since that's no longer an attr option?

Reply:
@rebeccabilbro - that makes sense since we are using colors.
@mgarod - I was reviewing your changes and they looked good.  I was working on fixing the merge conflicts - so I will make this change as well and push out the changes and merge it in.